### PR TITLE
Fixed PR #7

### PR DESCRIPTION
**Pull request by Ascor8522**
Fixed conflicts in https://github.com/InternationalScratchWiki/scratch-wiki.info/pull/7

* re-added wiki logos
* re-moved css to dedicated file
* now using scss for easier maintainability
* refractored code
    * easier php logic
    * now using html flexbox instead of hard-coded table
    * support for test wiki languages
* added some translations
    * de
    * nl
    * it
    * removed unnecessary "test"
* added URL language parameter

Preview of new home page:
_Font might not look the same, I have custom fonts installed_
image

On mobile:
(yes, I know, the page might be a bit long, need to find a better design)
image

### CODE REVIEW

**Comment by GrahamSH-LLK** [comment]
You could probably shrink each box into a strip on mobile.

**Comment by Ascor8522** [comment]
Made some changes and it already looks much cleaner.
Suggestions are always welcome.

image

File: assets/scss/style.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
+@use 'wiki.scss';
```
Comment (jacob-g) [inline assets/scss/style.scss:1]: My one concern here is that you used SCSS. What did you use to compile this to a regular CSS file? (Knowing what tool you used could enable us to use SCSS elsewhere and potentially automate the CSS compilation)

**Review by jacob-g — COMMENTED** [review]
Other than my comment about SCSS, looks good.

**Pull request merged**